aspnet/security/authorization/policies.rst

```diff
@@ -63,7 +64,7 @@ A requirement doesn't need to have data or properties.
 Authorization Handlers
 ----------------------
 
-An authorization handler is responsible for the evaluation of any properties of a requirement. The  authorization handler must evaluate them against a provided ``AuthorizationContext`` to decide if authorization is allowed. A requirement can have :ref:`multiple handlers <security-authorization-policies-based-multiple-handlers>`. Handlers must inherit ``AuthorizationHandler<T>`` where T is the requirement it handles. 
+An authorization handler is responsible for the evaluation of any properties of a requirement. The  authorization handler must evaluate them against a provided :dn:class:`~Microsoft.AspNetCore.Authorization.AuthorizationHandlerContext` to decide if authorization is allowed. A requirement can have :ref:`multiple handlers <security-authorization-policies-based-multiple-handlers>`. Handlers must inherit :dn:class:`~Microsoft.AspNetCore.Authorization.AuthorizationHandler<TRequirement>` where ``TRequirement`` is the requirement it handles. 
```

Comment:
@Rick-Anderson looks good.  Correct, at least one handler must call Succeed() on every requirement, and Fail must not have been called.  Handlers don't return values though, they manipulate things via the context.  

We do have a somewhat new RequireAssertion which is intended for 'simplish' logic that doesn't care about a requirement, and that one takes a `Func<AuthorizationHandlerContext, bool>` which can be used to return true instead of defining a handler/overriding a method/Succeed.